python/tank/descriptor/io_descriptor/git.py

```diff
@@ -147,6 +147,14 @@ def _clone_then_execute_git_commands(self, target_path, commands):
             log.debug("Restoring cwd (to '%s')" % cwd)
             os.chdir(cwd)
 
+        # Cleanup the folder of any git related files.
+        shutil.rmtree(os.path.join(target_path, ".git"), ignore_errors=True)
```

Comment:
I'll revert the code and will update the `build_bundle_cache.py` script to cleanup the .git folders in the meantime.